docs/.gitignore

```diff
@@ -1 +1,2 @@
 _build
+source
```

Comment:
Could we change this from "source" to "source/apidoc". Just so we don't accidentally hide changes in the source root directory.